pkg/webhook/policy.go

```diff
@@ -235,6 +265,27 @@ func (h *validationHandler) getDenyMessages(res []*rtypes.Result, req admission.
 					"request_username", req.AdmissionRequest.UserInfo.Username,
 				).Info("denied admission")
 			}
+			if ref != nil && ref.Name == "gatekeeper-controller-manager" {
```

Comment:
What's the logic path we're using if the ref isn't to gatekeeper-controller-manager?

If this is future-proofing against namespace-scoped constraints, we can just add the logic in later (YAGNI principle)